units/system-update-cleanup.service

```diff
@@ -30,6 +30,9 @@ SuccessAction=reboot
 ConditionPathExists=|/system-update
 ConditionPathIsSymbolicLink=|/system-update
 
+ConditionPathExists=|/etc/system-update
+ConditionPathIsSymbolicLink=|/etc/system-update
+
 [Service]
 Type=oneshot
-ExecStart=rm -fv /system-update
+ExecStart=rm -fv /system-update /etc/system-update
```

Comment:
i hope "rm -fv" invoked on a path that doesn't exist put points into a read-only file system geneerates ENOENT rather than EROFS. Did you verify that? because if it does not then this here might generate an unpleasent log message.

Reply:
Because we are using "-f" we don't generate error messages of any type, everything is a success right? At least for any of the cases I've tested.

Reply:
right